.cypress/cypress/integration/around_filters.js

```diff
@@ -127,3 +127,84 @@ describe('Around page filtering and push state', function() {
         cy.should('not.contain', 'Back to all reports');
     });
 });
+
+describe('Accessing around page filters on small screens', function() {
+    describe('When I search for a postcode on a small screen', function() {
+        before(function() {
+            cy.viewport(375, 667);
+            cy.visit('/');
+            cy.get('[name=pc]').type(Cypress.env('postcode'));
+            cy.get('[name=pc]').parents('form').submit();
+        });
+
+        // Work around viewport bug https://github.com/cypress-io/cypress/issues/1534
+        // Manually reset the viewport before each it() and then wait
+        // a little while for the viewport resize to take effect.
+        beforeEach(function(done) {
+            cy.viewport(375, 667);
+            setTimeout(done, 500);
```

Comment:
I should probably use `cy.wait(500)` here instead. Then there’ll be no need for the `done` callback.